google/cloud/bigtable/table_readmodifywriterow_test.cc

```diff
@@ -270,7 +274,8 @@ TEST_F(TableReadModifyWriteTest, UnrecoverableFailureTest) {
       .WillRepeatedly([](grpc::ClientContext* context,
                          google::bigtable::v2::ReadModifyWriteRowRequest const&,
                          google::bigtable::v2::ReadModifyWriteRowResponse*) {
-        EXPECT_STATUS_OK(IsContextMDValid(
+        ::google::cloud::testing_util::ValidateMetadataFixture fixture;
```

Comment:
here, I thought we'd use the member variable `validate_metadata_fixture_`. (again, I don't care. I am going to stop commenting on these things)

Reply:
You are right, fixed.